File: python/1038.py
cod,  qtd = map(int,input().split())

if cod == 1:
    preco = 4.0
    precoTotal = (float (preco * qtd)) 
elif cod == 2:
    preco = 4.50
    precoTotal = (float (preco * qtd)) 
elif cod == 3:
    preco = 5.0
    precoTotal = (float (preco * qtd)) 
elif cod == 4:
    preco = 2.0
    precoTotal = (float (preco * qtd)) 
elif cod == 5:
    preco = 1.50
    precoTotal = (float (preco * qtd)) 

print('Total: R$ {:.2f}'.format(precoTotal))

File: c/1038.c
#include <stdio.h>

int main(){
    int cod, qtd;
    float preco, contaFinal;

    scanf("%d%d", &cod, &qtd);

    switch (cod)
    {
    case 1 :
        preco = 4.;
        contaFinal = preco * qtd;
        break;
    case 2 :
        preco = 4.50;
        contaFinal = preco * qtd;
        break;
    case 3 :
        preco = 5.;
        contaFinal = preco * qtd;
        break;
    case 4 :
        preco = 2.;
        contaFinal = preco * qtd;
        break;
    case 5 :
        preco = 1.50;
        contaFinal = preco * qtd;
        break;   
    default:
        break;
    }

    printf("Total: R$ %.2f\n",contaFinal);


    return 0;
}